crates/holochain/src/conductor/kitsune_host_impl/query_region_set.rs

```diff
@@ -15,6 +21,15 @@ pub async fn query_region_set(
 ) -> ConductorResult<RegionSetLtcs> {
     let (arq_set, rounded) = ArqBoundsSet::from_dht_arc_set_rounded(&topology, strat, &dht_arc_set);
     if rounded {
+        // If an arq was rounded, emit a warning, but throttle it to once every LOG_RATE_MS
+        // so we don't get slammed.
+        let _ = LAST_LOG_MS.fetch_update(Ordering::Relaxed, Ordering::Relaxed, |t| {
+            let now = Timestamp::now();
+            now.checked_difference_signed(&Timestamp::from_micros(t * 1000))
+                .map(|d| d > chrono::Duration::milliseconds(LOG_RATE_MS))
+                .unwrap_or(false)
+                .then(|| now.as_millis())
+        });
```

Comment:
am i missing something? I don't see how this affects limiting the logging.

Reply:
> Note: This may call the function multiple times if the value has been changed from other threads in the meantime, as long as the function returns Some(_), but the function will have been applied only once to the stored value.

Maybe a bit pedantic because there's probably not a lot of contention here, but you may not want to do the actual logging inside the callback because it could get erroneously triggered multiple times.

Reply:
That's true, I'll change this